README.adoc

```diff
@@ -268,26 +275,26 @@ and `system/src/main/liberty/config/server.xml` files:
 <feature>usr:opentracingZipkin-0.30</feature>
 ----
 
-Make sure your services are running, then simply point your browser to any of their endpoints and
+Make sure your services are running. Then simply point your browser to any of their endpoints and
 check your Zipkin server for traces.
 
 
 === Enabling explicit distributed tracing
 
-The `@Traced` annotation can be used to define explicit span creation for specific classes and methods.
-If the annotation is placed on a class, then the annotation is automatically applied to all methods
-within that class. If the annotation is placed on a method, then it overrides the class annotation
-if one exists.
+Use the `@Traced` annotation to define explicit span creation for specific classes and methods.
+If you place the annotation on a class, then its automatically applied to all methods within that class.
+If you place the annotation on a method, then it overrides the class annotation if one exists.
 
-The `@Traced` annotation can be configured with two parameters:
+The `@Traced` annotation can be configured with the following two parameters:
 
-- `value=[true|false]` (true by default), indicates whether or not a particular class or method should
-be traced. For example, while all JAX-RS methods are traced by default, you can disable tracing using
-the `@Traced(false)` annotation.
-- `operationName=<Span name>` indicates the name of the span assigned to the particular method being
-traced. If omitting this parameter, the span will be named in form `<package name>.<class name>.<method name>`.
-If placed on a class, all methods within that class will have the same span name unless explicitly
-overwritten with another `@Traced` annotation.
+- The `value=[true|false]` parameter indicates whether or not a particular class or method should
+be traced. For example, while all JAX-RS methods are traced by default, you can disable tracing by using
+the `@Traced(false)` annotation. This parameter is set to true by default.
+- The `operationName=<Span name>` parameter indicates the name of the span that is assigned to the
+particular method being traced. If you omit this parameter, the span will be named with the following
```

Comment:
What about..."...particular method **that is** traced."?

Reply:
Ok I'm good with that